xla/service/gpu/gemm_algorithm_picker_test.cc

```diff
@@ -67,6 +71,18 @@ class GemmAlgorithmPickerTest : public HloTestBase,
   }
 };
 
+TEST_P(GemmAlgorithmPickerTest, BlasGetVersion) {
+#if TENSORFLOW_USE_ROCM && TF_ROCM_VERSION < 60200
+  GTEST_SKIP() << "This API is not available on ROCM 6.1 and below.";
+#endif
```

Comment:
This is definitely possible. E.g.:

https://github.com/openxla/xla/blob/main/xla/service/gpu/ir_emitter_triton_test.cc#L1439

Reply:
Yes, DeviceDescription can provide us access to GpuComputeCapability, hence we can distinguish between CUDA and ROCM (and also between individual gpu archs on these platforms). But, unfotunately, we are not able to obtain HIP or CUDA runtime version from there, that is why, other tests also have to use macros, e.g. [gemm_rewrite_test](https://github.com/openxla/xla/blob/5f7bb7990c17187901ae4fb6db3dce3287fdf296/xla/service/gpu/tests/gemm_rewrite_test.cc#L4988-L4994). 

I guess, this can be implemented by filling in some constant in cuda- and rocm-specific files from the respective macro and then adding that constant to DeviceDescription upon initialization.

Reply:
What about `DeviceDescription::runtime_version()`?

https://github.com/openxla/xla/blob/6deb4620842bce97e5bc04584d77fab6a41d9136/xla/stream_executor/device_description.h#L262

Reply:
oh ok, it seems to be already there, let me check it if works fine for rocm

Reply:
This variable seems not to be filled properly on ROCM platform, so I have added it. So, now we can avoid macros in this test